php-scoper.phar.pubkey

```diff
@@ -0,0 +1,14 @@
+-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAxKi/ABhiCMn7hqH9/W0x
```

Comment:
PHP Scoper should be a dependnecy in `composer.json`

Reply:
In docs it is not recommended.

Reply:
Still, should be dependency in `composer.json`. Without that we have to maintain phar manually.

What advantages do you see?

Reply:
Conflict avoidance.

Reply:
I just tried `bamarni/composer-bin-plugin` but it cannot work due to php-parser 3/4 conflict.

![image](https://user-images.githubusercontent.com/924196/39951250-65585e94-5588-11e8-9eb6-67d2b0341e6a.png)

Reply:
So phar seems as the best option for the time being. It doesn't evolve much and the php-parser 4 + better reflectoin is main blocker there, so missing update-ability won't be such a problem.